package.json

```diff
@@ -26,7 +26,7 @@
     "passport-strategy": "^1.0.0",
     "uid2": "0.0.x",
     "valid-url": "^1.0.9",
-    "xml-crypto": "auth0/xml-crypto#fix-digest",
+    "xml-crypto": "^2.0.0",
```

Comment:
Does this push us further away from upstream?

Is there a PR to make this same change (and the others we've modified like xml-encryption) in that repo?

Reply:
It does slightly. I'll see what happens when I mimic the upstream, but I suspect audit will flag the xml-crypto.

Reply:
Actually, the upstream reference still produces this effect:
`$ npm audit
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "xml-crypto@^2.0.0": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lc/.npm/_logs/2021-02-01T14_54_58_509Z-debug.lo`

Reply:
**Update**
The above is OBE. Issue was fixed in saml-proxy.